PowerToys.sln

```diff
@@ -2220,18 +2222,6 @@ Global
 		{D962A009-834F-4EEC-AABB-430DF8F98E39}.Release|x64.Build.0 = Release|x64
 		{D962A009-834F-4EEC-AABB-430DF8F98E39}.Release|x86.ActiveCfg = Release|x64
 		{D962A009-834F-4EEC-AABB-430DF8F98E39}.Release|x86.Build.0 = Release|x64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|ARM64.ActiveCfg = Debug|ARM64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|ARM64.Build.0 = Debug|ARM64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|x64.ActiveCfg = Debug|x64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|x64.Build.0 = Debug|x64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|x86.ActiveCfg = Debug|x64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Debug|x86.Build.0 = Debug|x64
-		{D9C5DE64-6849-4278-91AD-9660AECF2876}.Release|ARM64.ActiveCfg = Release|ARM64
```

Comment:
Feeling in my head

Reply:
I'm not sure that's the issue - ```D9C5DE64-6849-4278-91AD-9660AECF2876``` is the project guid for ```MouseJumpUI.UnitTests``` - see https://github.com/microsoft/PowerToys/blob/3b7adbe6acc381d11d8e17866da4b79444679c08/src/modules/MouseUtils/MouseJumpUI.UnitTests/MouseJumpUI.UnitTests.csproj#L8

That was one other thing I changed as part of moving some code to ```MouseJump.Common``` - I moved the test code to a new ```MouseJump.Common.UnitTests``` project...

Reply:
I think it's pointed me in the right direction though. It looks like for whatever reason the project GUIDs are wrong in the solution file compared to the one specified in the *.csproj files.

I've just removed and re-added the projects to the solution and the diff looks like this:

```
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MouseJump.Common", "src\modules\MouseUtils\MouseJump.Common\MouseJump.Common.csproj", "{F35E1EF3-C2AE-4407-8029-92B8F2B4A382}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MouseJump.Common", "src\modules\MouseUtils\MouseJump.Common\MouseJump.Common.csproj", "{923DF87C-CA99-4D1C-B1D2-959174E95BFA}"

-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MouseJump.Common.UnitTests", "src\modules\MouseUtils\MouseJump.Common.UnitTests\MouseJump.Common.UnitTests.csproj", "{6A10C2B0-92AE-4928-996B-FCF55D2E3F39}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MouseJump.Common.UnitTests", "src\modules\MouseUtils\MouseJump.Common.UnitTests\MouseJump.Common.UnitTests.csproj", "{D5E42C63-57C5-4EF6-AECE-1E2FCA725B77}"
```

I'll try committing that shortly and fingers crossed it'll fix the build error...

Reply:
So that didn't work :-(

Reply:
The latest commit (adding ```<RuntimeIdentifiers>win-x64;win-arm64</RuntimeIdentifiers>``` to the *.csproj seems to have fixed the NETSDK1047 error (more by luck than judgement I feel), but I'm still seeing failures from the ```verifyDepsJsonLibraryVersions.ps1``` build step.

Will continue to investigate...